MM-30544. Changed email references.

rachelbritton opened the pull request:
Changed all menandmice.com email references to support@bluecatnetworks.com. Also fixed linking and grammar issues in affected files.

rachelbritton committed:
MM-30544. Changed email references.

Changed all menandmice.com email references to support@bluecatnetworks.com. Also fixed linking and grammar issues in affected files.

agudgeirsson commented on guides/reference/health_bar_information.rst:
```diff
@@ -43,7 +43,7 @@ This error is displayed when the defined failover partner server for a DHCP serv
 """"""""""""""""""
 
-This warning is shown when DNS server (e.g. BIND) is unable to load its zones. This warning is also posted if the server has not successfully finished checking all zones for errors.
+This warning is shown when DNS server (e.g., BIND) is unable to load its zones. This warning is also posted if the server has not successfully finished checking all zones for errors.
```
Additional ,

rachelbritton replied:
It took me until now to actually look into this. The comma is used after e.g. in American English but not in British English. So I think it should be there?

agudgeirsson replied:
Ok, I understand. I will let you decide this, maybe check with the docs team what is the policy in general regarding this but I will approve whatever you want to do

rachelbritton replied:
Confirmed with Docs team that we should use commas for any instances of i.e. or e.g..

agudgeirsson commented on guides/reference/health_bar_information.rst:
```diff
@@ -55,8 +55,6 @@ Failed to load ODBC Driver for SQL Server
 
 The preferred version of the Microsoft ODBD Driver for SQL Server failed to load. The latest version can be downloaded from `https://www.microsoft.com/en-us/download/details.aspx?id=53339 <https://www.microsoft.com/en-us/download/details.aspx?id=53339>`_.
 
-Failover partner Down
```
Why is this being removed ?

rachelbritton replied:
Adding them back in. I had removed them here because there was no content/context around them, but instead I would rather get info and create a new issue to add it.

agudgeirsson commented on guides/reference/health_bar_information.rst:
```diff
@@ -66,13 +64,8 @@ This warning is shown when the active Central server went down and a standby Cen
-""""""""""""""""""""""""""""
+This warning is shown when the current license for a Men&Mice component has been exceeded. For example, if the appliance license is only valid for a single appliance, and another appliance is added, then the additional appliance is not shown. Please contact `support@bluecatnetworks.com <mailto:support@bluecatnetworks.com>`_ to resolve the matter.
 
-Orphaned PTR records in zone
```
Why is this being removed ?

rachelbritton replied:
See comment above

agudgeirsson commented on guides/reference/health_bar_information.rst:
```diff
@@ -102,28 +95,26 @@ This warning is shown when one of the following situations occur:
 """"""""""""""""""""""""""""""""""""""""""""""""""""""
 
-This warning is shown when Men&Mice Central can’t connect to the DHCP Server Controller service to check whether scope contains inconsistencies. The reason could be if the DHCP Server Controller is offline or not turned on.
+This warning is shown when Men&Mice Central can’t connect to the DHCP Server Controller service to check whether a scope contains inconsistencies. The reason could be if the DHCP Server Controller is offline or not turned on.
```
Micetro Central

rachelbritton committed:
MM-30544. Fixed grammar and naming.

Updated naming conventions and fixed grammar.

Added removed section headers back in.

rachelbritton committed:
MM-30544. Updated terminology.

Noticed several more terms that needed updating.

rachelbritton committed:
Merge branch 'latest' into issues/MM-30544-change-email-references
